src/components/Home/Diagram/index.tsx

```diff
@@ -2,9 +2,9 @@ import React, { forwardRef } from 'react'
 import Slider from 'react-slick'
 import cn from 'classnames'
 
-import LayoutWidthContainer from 'gatsby-theme-iterative-docs/src/components/LayoutWidthContainer'
-import ShowOnly from 'gatsby-theme-iterative-docs/src/components/ShowOnly'
-import Link from 'gatsby-theme-iterative-docs/src/components/Link'
+import LayoutWidthContainer from '@dvcorg/gatsby-theme-iterative/src/components/LayoutWidthContainer'
```

Comment:
Text isn't arranged right in the desktop version: 

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/43496356/170359799-8cfdda24-12db-44f6-b019-7ec8827bf4cc.png">

Reply:
I don't get the issue on Firefox, but I do on Chrome. Seems to just be a simple flexbox issue because the columns are so close to 100%- if you set `box-sizing: border-box` on the `column` class it resolves the issue.

Reply:
Ah yeah, and the rule is already there in the stylesheet but not applying because of some weird postcss stuff

Reply:
By using https://github.com/iterative/gatsby-theme-iterative/pull/20 to disable tailwind, the issue is fixed. I think it can serve as a band-aid while we try to get a more real fix in.

Reply:
> I don't get the issue on Firefox, but I do on Chrome. Seems to just be a simple flexbox issue because the columns are so close to 100%- if you set `box-sizing: border-box` on the `column` class it resolves the issue.

Yes, it could be solved by overriding the CSS.

> Ah yeah, and the rule is already there in the stylesheet but not applying because of some weird postcss stuff

But, yes the real issue was with `postcss-nested` not working properly.